etc/pg/postgresql.conf

```diff
@@ -61,7 +61,7 @@ listen_addresses = '*'		# what IP address(es) to listen on;
 					# defaults to 'localhost'; use '*' for all
 					# (change requires restart)
 #port = 5432				# (change requires restart)
-max_connections = 100			# (change requires restart)
+max_connections = 200			# (change requires restart)
```

Comment:
Informational - osidb stage/prod are either 3000/5000 max conns ... though it is good to constrain this further for local development